### ISSUE TITLE
The button to remove a file is not accessible to keyboard users

### ISSUE THREAD

**Issue by krismadden** [vugar005/ngx-awesome-uploader#126]
Anchor tags without an href are not in the taborder and need explicit keyboard events. However, buttons do not. Is it possible to change the anchor tag to a button tag to solve this?
